lightbluetent/templates/society/begin_session.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<form method="post">
+
+<input type="hidden" name="_csrf_token" value="{{ csrf_token() }}">
+
+<div class="form-row">
+    <div class="col-md-6 px-5">
```

Comment:
Is there a reason why we're creating a column half the width of the screen here and not just filling the whole screen?